src/test/java/tests/SortQualifierTests.java

```diff
@@ -397,4 +399,34 @@ public void statusOrdering() {
                 .map(TurboIssue::getId)
                 .collect(Collectors.toList()));
     }
+
+    @Test
+    public void stability() {
```

Comment:
It was to address a coverage drop. Not exactly sure why it happens, but the build has only passed twice over the past couple of days, and both times showed some line in the sorting code as not being covered.